t/t3428-rebase-edit-todo.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test_description='rebase should reread the todo file if an exec modifies it'
+
+. ./test-lib.sh
+
+todo=$(git rev-parse --git-dir)/rebase-merge/git-rebase-todo
```

Comment:
I don't know what --git-dir returns, since I'm not terrible familiar with the test harness. If it's possible that --git-dir returns something with spaces then what I had was actually correct: she'll variable assignment is fully parsed before expansion, so that even if `$bar` has a space in its expansion, `foo=$bar` will still assign the whole thing without quotes. When passing to a redirection operator, however, the quotes are required.

But if we can always just assume `.git` is valid then hardcoding makes the most sense, and obviates the need for any of this.